From 6630f0550ff12330406900d85a61b4ce449cbaa8 Mon Sep 17 00:00:00 2001
From: Dan Schatzberg <schatzberg.dan@gmail.com>
Date: Wed, 10 May 2023 11:42:56 -0700
Subject: [PATCH 38/39] atropos: Remove static libclang dependency

Most distributions provide a shared lib version of libclang, so building
atropos is not as easy as it should be. #6 is an example of this being
confusing to other developers.

We have no particular dependency on libclang being static, so this can
be configured more conventionally.

Signed-off-by: Dan Schatzberg <schatzberg.dan@gmail.com>
---
 tools/sched_ext/atropos/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/sched_ext/atropos/Cargo.toml b/tools/sched_ext/atropos/Cargo.toml
index 7462a836d..1e47d86fc 100644
--- a/tools/sched_ext/atropos/Cargo.toml
+++ b/tools/sched_ext/atropos/Cargo.toml
@@ -21,7 +21,7 @@ ordered-float = "3.4.0"
 simplelog = "0.12.0"
 
 [build-dependencies]
-bindgen = { version = "0.61.0", features = ["logging", "static"], default-features = false }
+bindgen = { version = "0.61.0" }
 libbpf-cargo = "0.13.0"
 
 [features]
-- 
2.40.1.445.gf85cd430b1

